test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -474,17 +501,17 @@ describe( `[${ host }] Calypso Gutenberg Site Editor Tracking: (${ screenSize })
 					return this.skip();
 				}
 
-				// First reset and save so we know test changes will trigger save updates.
+				// Reset global styles before testing.
 				await clickGlobalStylesResetButton( this.driver );
-				await editor.saveAll();
+				await saveGlobalStyles( this.driver );
 				// Wait for debounce and entity fetching before clearing events.
 				await this.driver.sleep( 500 );
 				await clearEventsStack( this.driver );
 
 				await changeGlobalStylesFontSize( this.driver, '11' );
 				await changeGlobalStylesColor( this.driver, 1, 1 );
 				await changeGlobalStylesColor( this.driver, 3, 1 );
-				await editor.saveAll();
+				await saveGlobalStyles( this.driver );
 				// Wait for debounce and entity fetching before accessing events stack.
 				await this.driver.sleep( 500 );
```

Comment:
Same here - it looks like a pattern that should be handled inside the `saveGlobalStyles` method.

Reply:
Makes sense, I moved that sleep into `saveGlobalStyles` for now.